common/types/shared.go

```diff
@@ -7842,3 +8095,18 @@ type RespondCrossClusterTasksCompletedResponse struct {
 type StickyWorkerUnavailableError struct {
 	Message string `json:"message,required"`
 }
+
+// SerializeRequest Serialize an arbitrary request for logging
+// pass in a pointer as a parameter to save space
+func SerializeRequest(request interface{}) (string, error) {
+	if request == nil {
+		return "", nil
+	}
+
```

Comment:
```suggestion
```
probably just leftovers from trying the cleanup

Reply:
done